### ISSUE TITLE
Upgrade to 17.09.1-ce-win42 hangs

### ISSUE THREAD

**Issue by evil-shrike** [docker/for-win#1417]
I accepted a prompt to upgrade Docker but the installer hangs.

![image](https://user-images.githubusercontent.com/1203230/33879471-8febf982-df3f-11e7-9bd1-8b259f3a15cd.png)

Canceling doesn't work. The dialog is just hangs.

**Comment by boskovisch** [comment]
Same stuff here, hangs on the Stable and Edge regardless, tried compatibility mode as well, same result.

**Comment by bjoernmartin** [comment]
Same here. Both upgrade from 17.09 to 17.09.1 ce win42 as well as a fresh install hang. 17.09 was installing + working fine.

Windows 10 Pro, version 10.0.16299.125

**Comment by boskovisch** [comment]
Tried this, still doesn´t work, it´s still stuck. I can go to the docker folder on program files, there are some files there, I can start docker from there, but it complains about the service being missing.

**Comment by Lucas-C** [comment]
I'm affected too, attempting to reinstall Docker4Windows after a fully uninstalling it and facing this bug: https://github.com/docker/for-win/issues/1555

**Comment by svassaux** [comment]
It's still hanging even after removing all docker files of my system

**Comment by boskovisch** [comment]
Didn´t work anyway for me, tried everything, cleaned registries and so, still the same issue. Installed on two other machines without problems. 

My suggestion is to have better logs and a verbose mode, in most situation we don´t need to know whats´going on under the hood, but in this case I needed to know the origin of the issue, or at least have some clue. 

I had virtualbox installed in the old machine, so that may be the reason, but even after uninstalling it and removing its traces it didn´t work.

**Comment by evil-shrike** [comment]
You won't believe but it hangs again, now upgrading to 17.12.0-ce-win46:
![image](https://user-images.githubusercontent.com/1203230/35044191-8204b958-fba0-11e7-8bb6-ba4f9fc44159.png)

**Comment by dazinator** [comment]
I saw an identical issue.
I had to reboot, then the old docker started and told me the backend wasn't installed! I had to completely uninstall docker, then install docker from fresh and that worked.

One thing I noticed, when this happened I was low on disk space. I freed up some space before I attempted the fresh install.

Could it be related to low disk space? As can be seen [from my screenshot](https://github.com/docker/for-win/issues/1902) it seems to have hung at an identical place to @evil-shrike  screenshot above - when extracting files.
